test/test_parse_osmchange_input.cpp

```diff
@@ -906,99 +500,69 @@ void test_large_message() {
 
   s << "</osmChange>";
 
-  // Valid message
-  try {
-    process_testmsg(s.str());
-  } catch (http::exception &e) {
-    throw std::runtime_error("test_large_message::001: Unexpected Exception");
-  }
+  REQUIRE_NOTHROW(process_testmsg(s.str()));
 }
 
-void test_object_limits() {
 
+// OBJECT LIMIT TESTS
+
+TEST_CASE("Create node, tags < max tags", "[osmchange][node][xml]") {
   auto test_settings = std::unique_ptr<global_settings_test_class>(new global_settings_test_class());
   test_settings->m_element_max_tags = 5000;
```

Comment:
By the way, thanks to catch2 it's fairly obvious now that this test case is by far the slowest one in this file:

```
test/test_parse_osmchange_input -D 1
7.754 s: Create node, tags < max tags
===============================================================================
All tests passed (7859 assertions in 73 test cases)

```